docs/infrastructure/logging.md

```diff
@@ -257,6 +264,116 @@ determines how often logger should send its collected events to sentry.io
 service. If `debug` value is `true`, logging will print to stdout additional
 trace information of the logging process itself.
 
+## Logging server
+
+The centralised logging service is implemented as a logging server started by
+the
+[Project Manager](https://github.com/enso-org/enso/blob/c47dba1d108e0d52e401333d1b6f6f4182436aa7/lib/scala/project-manager/src/main/scala/org/enso/projectmanager/boot/ProjectManager.scala#L326-L345).
+The implementation of logging server is in
+[org.enso.logging.service.logback.LoggingServer](https://github.com/enso-org/enso/blob/da6a6ae9dde9c0cef02c1d07be5e03660b40d086/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/LoggingServer.java).
+The logging server
+[listens](https://github.com/enso-org/enso/blob/da6a6ae9dde9c0cef02c1d07be5e03660b40d086/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/SocketServer.java#L56)
+to clients that connect via a socket. The client is, for example, language
+server that is started as a subprocess by the project manager. The clients use
+[org.enso.logging.service.logback.DeferredProcessingSocketAppender](https://github.com/enso-org/enso/blob/566d3d503e35ccf8c363facd590d1332683bce3a/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/DeferredProcessingSocketAppender.java)
+to send log events to the logging server. The logging server then properly
+dispatches the received logging event to all the appenders in
+[SocketLoggingNode](https://github.com/enso-org/enso/blob/3e0b4dd7413e45373dc4ac26f124ee6aedffd50c/lib/scala/logging-service-logback/src/main/java/org/enso/logging/service/logback/SocketLoggingNode.java#L87-L94).
```

Comment:
This describes how the logging works in the IDE, right?
- we should stress that this section talks about the IDE mode
- How does that work in the CLI mode? Is it different?

Reply:
Good point. Described in 40b6ac71e98de8626c05d1822ac5bc3eab9626af